blog/_posts/2020-05-26-create-your-own-theme-detailed-walkthrough.md

```diff
@@ -0,0 +1,372 @@
+---
+layout: post
+title: Create your own theme in minutes !
+---
+
+In Convos [v4.xx](https://convos.chat/blog/2020/5/14/theming-support-in-4-point-oh.html) it is super easy to customize
+or create a theme.  Follow this detailed walkthrough where I create a **"hack3r"** theme ! 
+
+<!--more-->
+
+
+## Smooth start
+
+You have to connect to your server where convos is running or you can do this tutorial locally : 
+
+`cd $CONVOS_HOME/public/themes`
+
+Then copy past an existing theme :
+
+`cp high-contrast_color-scheme-light.css mytheme.css`
+
+The name of your file is not important, the theme name will be included in the header of the CSS file.
+
+`vi mytheme.css`
```

Comment:
"vi" might not be familiar to people. I tend to use `$EDITOR mytheme.css` instead, but "nano" or something simple would also work.